VideoPlusChat/src/main/java/com/microsoft/device/display/samples/videochatcomposesample/ui/views/MainPage.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.videochatcomposesample.ui.views
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusManager
+import androidx.compose.ui.platform.LocalFocusManager
+import com.microsoft.device.display.samples.videochatcomposesample.models.InfoProvider
+import com.google.android.exoplayer2.ExoPlayer
+import com.microsoft.device.dualscreen.twopanelayout.TwoPaneLayout
+import com.microsoft.device.dualscreen.windowstate.WindowState
+
+var infoProvider: InfoProvider = InfoProvider()
+
+@Composable
+fun MainPage(windowState: WindowState, player: ExoPlayer) {
+
+    // very strange bug with Exoplayer and Windowstate so I have to define them here or else the player will not show
```

Comment:
I think we should try to find the root cause of the issue. Currently, the video can be played in the single-screen mode with these extra definitions, but it can't be played(black screen with audio) when the app is spanned to the dual-portrait mode when I was testing.